### ISSUE TITLE
Menu over MKMapView swipe

### ISSUE THREAD

**Issue by dennytwix** [dekatotoro/SlideMenuControllerSwift#336]
When I try to swipe from the edge of the screen, to show the menu, map is scrolling too. 
It's not possible to use addPriorityToMenuGesuture, since it accepts only scroll view.

**Comment by abhijithExperion** [comment]
@dennytwix How are you showing menu on the view controllers which is not pushed but instead presented? Any solution?

**Comment by dennytwix** [comment]
Sorry, I am not sure that understood your question. It seems that it is not related to my issue.

**Comment by iDevPro** [comment]
Not only MKMapView, also UIPageViewController has activity on pan gesture from edge :)

**Comment by dennytwix** [comment]
@iDevPro yes, basically any component which has pan gesture and which is not UIScrollView. 
But it's easy to extract pan gesture from page controller and require it to fail :)

**Comment by iDevPro** [comment]
I'm already solve it by fail gestures :)

**Comment by ssowri1** [comment]
Yes. I am also facing kind of issues when I am try to open the menu from edge my map view also scrolling. How to restrict this. Can you share your solution?

**Comment by dennytwix** [comment]
@ssowri1  unfortunately we just disabled opening menu with edge gesture :(

**Comment by ssowri1** [comment]
@dennytwix Thank you for your response. 

Yes. I also tried to disable the edge gesture along with below codes. But still its working the same. can you please share the code to disable the edge gesture.

            SlideMenuOptions.tapGesturesEnabled = true
            SlideMenuOptions.panGesturesEnabled = false

**Comment by dennytwix** [comment]
@ssowri1 `SlideMenuOptions.simultaneousGestureRecognizers = false`

**Comment by ssowri1** [comment]
@dennytwix Thank you. It's working fine :-)
